build.sh

```diff
@@ -33,6 +33,8 @@
 # Note: this build does not sign the binaries
 # It's up to the consumer of the binaries to sign them
 
+# Calling with --skip-snk will skip applying strong names to Assembies.
```

Comment:
L missing in assemblies.

Reply:
Thank you, I have corrected it in a new commit.